OffchainLabs/arb-token-bridge
# refactor: create a consistent loader atom

### Summary
Currently we are using `react-loader-spinner` with custom styles in all files wherever we need a loader. With this PR we are making it consistent by creating a common Loader component.

### Steps to test
Functionally it changes nothing. 
Nothing should look awkward in the places, where the `Loader` has been updated, listed as follows:
- Main (large) App loader while connecting to the network.
- Inside the connect wallet button while connecting to networks.
- Transactions table while loading withdrawals
- Deposit and Withdrawal cards which come when we initiate transactions (successful state)
- Token import dialog while importing token
- Searching for tokens in the Token dropdown
- Transfer Tokens' Panel and Input wherever fetching is in progress
